public/content/lessons/2018/sphere-area/index.mdx

```diff
@@ -6,4 +6,421 @@ video: GNcFjFmqEc8
+
+<Figure video="rotation-animation.mp4" />
+
+</Accordion>
+
+Now, if you’re really thinking critically, you might still not be satisfied that this shows that the surface area of the sphere equals the area of this cylinder label. After all, these little rectangles only *approximate* the relevant areas.
+
+Well, the idea is that this approximation gets closer and closer to the true value for finer and finer coverings. Since for any specific covering, the sphere rectangles have the same area as the cylinder rectangles, whatever values each of these two series of approximations are approaching must actually be the same.
+
+<Figure image="equal-in-limit.png" />
+
+I mean, as you get really aggressively philosophical about what we even mean by surface area, these sorts of rectangular approximations and not just aids in our problem-solving toolbox, they end up serving as a way of rigorously defining the area of smooth curved surfaces.
+
+<Figure image="surface-area-definition.png" />
+
+This kind of reasoning is essentially calculus, just stated without any of the jargon. In fact, I think neat geometric arguments like this, which require no background in calculus to understand, can serve as a great way to tee things up for new calculus students so that they have the core ideas before seeing the definitions which make them precise (rather than the other way around).
+
+<Figure
+  image="preparing-for-calculus.png"
+  caption="These geometric arguments are great preparation for understanding calculus."
+/>
+
+### Connection to circles
+So as I said before, if you’re itching to see a direct connection to four circles, one nice way is to unwrap these circles into triangles. If this is something you haven’t seen before, I go into much more detail about why this works in <LessonLink id="essence-of-calculus">the first lesson of the calculus series</LessonLink>.
+
+The basic idea is to relate thin concentric rings of the circle with horizontal slices of this triangle.
+
+<Figure
+  image="ring-to-slice.png"
+  caption="Each circle is unwrapped such that a thin ring of the circle corresponds to a thin line of the triangle."
+/>
+
+Because the circumference of each such ring increases linearly in proportion to the radius, always equal to $2\pi r$, when you unwrap them all and line them up, their ends will form a straight line (as opposed to some kind of curvey shape), giving you a triangle with a base of $2\pi r$ and a height of $r$.
+
+And four of these unwrapped circles fit into our rectangle, which is in some sense an unwrapped version of the sphere’s surface.
+
+<Figure image="triangles-fit-rectangle.png" />
+
+## Second approach
+Nevertheless, you might wonder if there’s a way than this to relate the sphere directly to a circle with the same radius, rather than going through this intermediary of the cylinder.
```

Comment:
```suggestion
Nevertheless, you might wonder if there’s a way to relate the sphere directly to a circle with the same radius, rather than going through this intermediary of the cylinder.
```